packages/core/integration-tests/test/cache.js

```diff
@@ -260,6 +243,20 @@ describe('cache', function() {
                 }),
               );
 
+              // At the moment, adding a JS config requires restarting the process
+              // due to https://github.com/babel/babel/pull/12211. Simulate this
+              // by clearing the require cache for @parcel/transformer-babel and @babel/core.
```

Comment:
That PR got merged and released: https://github.com/babel/babel/releases/tag/v7.12.17
Does that change anything?

Reply:
I will test

Reply:
Yep, looks like this is unnecessary after the Babel upgrade!